<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>872</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,7,2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="spinTime">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Min(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkAutoScraping">
          <property name="text">
           <string>AutoScraping</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Last Updated Booking Number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtBookingNumber">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnScraping">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Start Scraping</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStop">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Stop Scraping</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnOpen">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Go Output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnExit">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Scraped Data</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="btnExport">
                <property name="text">
                 <string>Export CSV</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btnAddrValidation">
                <property name="text">
                 <string>1.Address Validation</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnMakeLetter">
                <property name="text">
                 <string>2.Making Letter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnMakeEnvelop">
                <property name="text">
                 <string>3.Making Envelop</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnMakePdfs">
                <property name="text">
                 <string>4.Print Pdfs</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnClearAll">
                <property name="text">
                 <string>5.Clear All</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableWidget" name="tblList">
              <column>
               <property name="text">
                <string>Booking number</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Fisrt Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Last Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Address1</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Address2</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>City</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>State</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Zipcode</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Correct Address</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Invalid Address</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QTableWidget" name="tblAddr">
            <column>
             <property name="text">
              <string>Booking number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Fisrt Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Last Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Address1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Address2</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>City</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>State</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Zipcode</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnClear2">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="edtConsole"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1293</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
